src/test/resources/s3-valid-inbound-cases.xml

```diff
@@ -19,7 +23,7 @@
 					bucket="TestBucket"
 					propertiesFile="testawscredentials.properties"
 					temporary-suffix=".temp"
-					local-directory-expression="@localDirectory"
+					local-directory="${foo.dir:/}"
```

Comment:
Yep. Have just wanted to ask you about the same :smile: 
OK. Let me grab it locally and test before merging.
Although I see already that `spring-cloud-aws-context-1.0` is redundant import. 
As well as I'm sure that this PP definition will work even without that `<context:property-placeholder>` declaration.
No?